src/agent0/ethpy/hyperdrive/state/pool_state.py

```diff
@@ -29,7 +29,9 @@ class PoolState:
     checkpoint_time: int
     checkpoint: CheckpointFP
     exposure: FixedPoint
-    variable_rate: FixedPoint
+    # TODO we may want to remove this from pool state and have users explicitly
```

Comment:
I think I like this (removing it from state; forcing explicit call) better, but if you'd rather punt that for a later time it's good with me.